mergin/client.py

```diff
@@ -758,6 +759,7 @@ def push_project(self, directory):
             return  # there is nothing to push (or we only deleted some files)
         push_project_wait(job)
         push_project_finalize(job)
+        self.clean_temp_files(directory)
```

Comment:
it would be better to clean temp files in the push_project_finalize() call instead of here...